### PR TITLE
Professional Services Playbook

### PR DESCRIPTION
**Pull request by ripixel**

### CODE REVIEW

File: professional-services/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Professional Services @ DVELP
+
+Our playbook for building bespoke client-solutions at DVELP.
+
```
Comment (tom-mullen) [inline professional-services/README.md:4]: @ripixelcodes let's remove these WIP parts, just start small and we can add in new parts after?

**Review by tom-mullen — CHANGES_REQUESTED** [review]

File: professional-services/interfaces.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+The Professional Services team interfaces with Finance and is responsible for providing them
+with:
+
+- Resource requirements
+- Annual Forecasts
+- Consumption reports for reconciliation
+
+## Marketing
+
+The Professional Services team interfaces with the Marketing team through dedicated Product
```
Comment (tom-mullen) [inline professional-services/interfaces.md]: Looks like you just copy pasted this?

File: professional-services/interfaces.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+## Sales & Account Management
+
+The Professional Services team interfaces with the Sales & Account Management organisation and
+is responsible for providing them with:
+
+- Assistance with scoping possible opportunities
+- Reviews of proposals
+- Accurate future capacity
+
+## Product
```
Comment (tom-mullen) [inline professional-services/interfaces.md:27]: Same

**Review by tom-mullen — CHANGES_REQUESTED** [review]

File: professional-services/roles/head-of-engineering.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+## Responsibilities
+
+The HoE takes full responsibility for the engineering processes associated with creating solutions, from proposal to go-live and ongoing support.
+
+### Uphold modern standards
+
+Many engineers have their own styles of coding and preferred development approaches. While for minutia this is acceptable, the HoE works with the engineers to create a set of standards to be used across multiple solutions where appropriate. This includes things such as:
+
+- Defacto libraries
```
Comment (tom-mullen) [inline professional-services/roles/head-of-engineering.md:13]: This list should be extended to include delivery processes. Let's try and make it as accurate/extensive as possible.

File: professional-services/roles/head-of-engineering.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+As the ways of doing things change, the engineering team will have to learn new skills, technologies, and patterns. The HoE is responsible for the processes allowing engineers to upskill themselves, and identifying situations when this must be done and ensuring the engineering team stay ahead of the curve.
+
+### Manage resourcing
+
+An engineering team can only be as effective as its headcount allows. The HoE works with the Head of Delivery to ensure utilisation is maximised, and to manage the recruitment and hiring of new team members when a need to do so is identified.
+
+### Be the interface to the engineering team
+
+The HoE will be the single point of contact for queries to/about the engineering team. This can take the form of [processes](#create-and-document-processes), but also in simply being available to field questions and participate in meetings where engineering input is required.
```
Comment (tom-mullen) [inline professional-services/roles/head-of-engineering.md]: Remove `simply`

File: professional-services/roles/head-of-engineering.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+### Be security-centric
+
+Our engineers must not only be security conscious, but have security front-of-mind when designing and delivering solutions. The HoE will be responsible for ensuring engineers are following correct security procedures, and that any solutions are security-centric.
+
+### Provide the right tools
+
+Good tool sets are an incredibly important part of allowing engineers to work at pace with minimal friction. The HoE will be responsible for ensuring the tools provided are relevant, and where one is not or does not exist, to procure one or to lead a team in building them.
```
Comment (tom-mullen) [inline professional-services/roles/head-of-engineering.md]: and where one is not or does not exist -> and where one does not exist

File: professional-services/roles/head-of-engineering.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+### Create and document processes
+
+The HoE will be responsible for enabling the engineering team to interface with the rest of the business in as frictionless a manner as possible. To this end they must create and document processes in partnership with other business functions that will allow this to happen; the end goal is that the HoE is not required to be "in the weeds" to solve business as usual tasks.
```
Comment (tom-mullen) [inline professional-services/roles/head-of-engineering.md]: The HoE will be responsible for enabling the engineering team to interface with the rest of the business with the least friction possible.

File: professional-services/roles/head-of-engineering.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+## Communication
+
+The HoE will have various stakeholders to communicate with.
```
Comment (tom-mullen) [inline professional-services/roles/head-of-engineering.md]: The HoE will have communicate regularly with the following stakeholders.

File: professional-services/roles/head-of-engineering.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+
+### Professional Services Head of Delivery
+
```
Comment (tom-mullen) [inline professional-services/roles/head-of-engineering.md:54]: You have overlooked the need to communicate with the delivery team (most important one!)

**Review by tom-mullen — CHANGES_REQUESTED** [review]

File: professional-services/interfaces.md (hunk 2 above)
Comment (ripixel) [reply professional-services/interfaces.md]: Yup, was wondering if you had any feedback on what the mechanism here will be? Or should I remove for now?

File: professional-services/interfaces.md (hunk 3 above)
Comment (ripixel) [reply professional-services/interfaces.md:27]: This I have modified for Professional Services relevance

File: professional-services/interfaces.md (hunk 2 above)
Comment (ripixel) [reply professional-services/interfaces.md]: Will remove for now.

File: professional-services/roles/head-of-engineering.md (hunk 9 above)
Comment (ripixel) [reply professional-services/roles/head-of-engineering.md:54]: D'oh!

File: professional-services/engineering/README.md [hunk 10]
```diff
@@ -0,0 +1 @@
+# Professional Services Engineering
```
Comment (tom-mullen) [inline professional-services/engineering/README.md]: Let's remove these for now

File: professional-services/delivery/README.md [hunk 11]
```diff
@@ -0,0 +1 @@
+# Professional Services Delivery
```
Comment (tom-mullen) [inline professional-services/delivery/README.md]: Remove for now

File: professional-services/interfaces.md [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+- [Support](#support)
+
+## Finance
+
+The Professional Services team interfaces with Finance and is responsible for providing them
+with:
+
+- Resource requirements
+- Annual Forecasts
+- Consumption reports for reconciliation
```
Comment (tom-mullen) [inline professional-services/interfaces.md:15]: Add in: Weekly utlisation and contribution reports

File: professional-services/roles/head-of-delivery.md [hunk 13]
```diff
@@ -0,0 +1,7 @@
+# Head of Delivery
```
Comment (tom-mullen) [inline professional-services/roles/head-of-delivery.md]: Remove for now and Annise can open a new PR

File: professional-services/roles/head-of-engineering.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
+# Head of Engineering
+
+The Head of Engineering (HoE) oversees the engineering team that creates solutions within the Professional Services team. They are ultimately responsible for the output from the engineering team, and support them in their business as usual functions. They are the arbiters of engineering standards and interfacing with the rest of the business.
```
Comment (tom-mullen) [inline professional-services/roles/head-of-engineering.md]: The Head of Engineering (HoE) oversees the engineering team in the delivery of bespoke, Professional Services solutions.

File: professional-services/roles/head-of-engineering.md [hunk 15]
```diff
@@ -0,0 +1,67 @@
...
+
+The HoE will be the single point of contact for queries to/about the engineering team. This can take the form of [processes](#create-and-document-processes), but also in being available to field questions and participate in meetings where engineering input is required.
+
+### Be security-centric
+
+Our engineers must not only be security conscious, but have security front-of-mind when designing and delivering solutions. The HoE will be responsible for ensuring engineers are following correct security procedures, and that any solutions are security-centric.
+
+### Provide the right tools
+
+Good tool sets are an incredibly important part of allowing engineers to work at pace with minimal friction. The HoE will be responsible for ensuring the tools provided are relevant, and where one does not exist, to procure one or to lead a team in building them.
```
Comment (tom-mullen) [inline professional-services/roles/head-of-engineering.md]: and where they do not exist, to procure them or to lead a team in building them.

File: professional-services/roles/head-of-engineering.md [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+
+### Create and document processes
+
+The HoE will be responsible for enabling the engineering team to interface with the rest of the business with the least friction possible. To this end they must create and document processes in partnership with other business functions that will allow this to happen; the end goal is that the HoE is not required to be "in the weeds" to solve business as usual tasks.
+
+## Communication
+
+The HoE will have communicate regularly with the following stakeholders.
```
Comment (tom-mullen) [inline professional-services/roles/head-of-engineering.md]: The HoE will communicate regularly with the following stakeholders:

File: professional-services/roles/head-of-engineering.md [hunk 17]
```diff
@@ -0,0 +1,67 @@
...
+
+As opportunities arise, the HoE must be able to identify resources to assist in the pre-sales and proposal processes.
+
+### Customers
+
+Interfacing directly with customers in order to provide support on whatever matters required, if their queries cannot be solved by the delivery team currently assigned to the project.
+
+## Reporting
+
+The HoE reports directly into the Chief Product Officer.
```
Comment (tom-mullen) [inline professional-services/roles/head-of-engineering.md]: Let's change this to the VP of Solutions.

**Review by tom-mullen — CHANGES_REQUESTED** [review]

**Pull request closed without merging**